backend/globaleaks/handlers/authentication.py

```diff
@@ -52,7 +52,7 @@ def login_whistleblower(store, receipt, using_tor2web):
     """
     login_whistleblower returns the WhistleblowerTip.id
     """
-    hashed_receipt = security.hash_password(receipt, GLSettings.memory_copy.receipt_salt)
+    hashed_receipt = security.hash_password(receipt, GLSettings.memory_copy.password_salt)
```

Comment:
Cause for the receipts(all, N) we are using the same salt given the fact that they do not have a username to which assign the salt, and given that using the same salt N times or N+1 times is exactly the same :)

Any suggestion?